pylmod/base.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/python
+"""
+ Python interface to Stellar Grade Book module
+
+ Defines the class StellarGradeBook
+"""
+
+import json
+import logging
+import requests
+
+
+log = logging.getLogger(__name__)  # pylint: disable=C0103
+
+
+class Base(object):
+    """
+    Python class representing interface to Stellar gradebook.
```

Comment:
All of this is wrong with this change, this should be the doc string for client maybe if you want to reuse, but this docstring should be something about how this is a base class that shouldn't be instantiated directly and a description of the base/common methods/attributes it provides for people wanting to extend it.